_includes/commands/table-cmds.md

```diff
@@ -4,6 +4,7 @@
 
 | Command                 | Description                                     |
 | ----------------------- | ----------------------------------------------- |
+| mceTableApplyCellStyle | Applies the specified styles to the selected cells. The following styles can be changed with this command: `background-color`, `border-color`, `border-style`, and `border-width`. Providing an empty value for a style will remove the style, such as `{ background-color: '' }`. {{site.requires_5_4v}} |
```

Comment:
Looks good. Where you have, `{ background-color: '' }`, I think it should be `{ 'background-color': '' }` to be valid JS.

Reply:
fixed in f80bc61fabd89b13c07894d33a0047e5e0680f45